### ISSUE TITLE
UIViewReportBrokenSuperviewChain on iOS 10

### ISSUE THREAD

**Issue by vitobellini** [zoonooz/ZFRippleButton#25]
I'm addressing a UIViewReportBrokenSuperviewChain issue which makes my app to crash.
My buttons are added on the view using Interface Builder, no code.
They were working fine on Xcode 7 / iOS 9. No changes are made with Xcode 8 / iOS 10, I just opened the project, run it on a real device and get this crash.

I'm using ZFRippleButton 0.5.1.

```
2016-10-21 10:02:48.620067 Cinemappy[9732:2494425] *** Assertion failure in void UIViewReportBrokenSuperviewChain(UIView *__strong, UIView *__strong, BOOL)(), /BuildRoot/Library/Caches/com.apple.xbs/Sources/UIKit/UIKit-3599.6.1/UIView.m:190
2016-10-21 10:03:05.951306 Cinemappy[9732:2494425] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'View has lost track of its superview, most likely through unsupported use of CALayer API on the view's layer. If this isn't a crash yet, it will be in the near future. 
    Problem view: <UIView: 0x15fc91740; frame = (0 0; 1000 1000); alpha = 0; layer = <CALayer: 0x1746298c0>>
    Expected parent: <ZFRippleButton.ZFRippleButton: 0x15fc90b40; baseClass = UIButton; frame = (0 0; 1000 1000); opaque = NO; autoresize = RM+BM; tintColor = UIExtendedSRGBColorSpace 1 1 1 1; layer = <CALayer: 0x174821f20>>
Break on UIViewReportBrokenSuperviewChain to debug.'
*** First throw call stack:
(0x1894dc1c0 0x187f1455c 0x1894dc094 0x189f66898 0x18f604f9c 0x18f605658 0x18fc7b390 0x189f122bc 0x189f07ab8 0x189f0765c 0x189f07444 0x18f42a76c 0x18f42a4d0 0x18f371be0 0x19553fd04 0x19553fad4 0x195553d5c 0x18f6195e8 0x18f33beac 0x1893c91a4 0x1893c90a0 0x18f32ac30 0x18f427330 0x18f61f2c4 0x18f32254c 0x18c7ea40c 0x18c7df0e8 0x18c7defa8 0x18c75bc64 0x18c
libc++abi.dylib: terminating with uncaught exception of type NSException
```

**Comment by vitobellini** [comment]
fix #23 works, I had to edit manually the file. May be I have to had an outdated repo of cocoapods.

**Comment by zoonooz** [comment]
Hi @vitobellini This was fixed in #23 and was waiting Cocoapods to fix lint bug. Now publishing 0.6
